authentication/templates/ui/dashboard.html

```diff
@@ -7,10 +7,17 @@
 <div style="text-align:center" class="col-sm-12 ">
 <h1>Welcome to your User Dashboard</h1>
 <h4>Someone need to format this so it doesnt suck.</h2>
+<form method="post" action="{% url 'account_logout' %}">
+    {% csrf_token %}
+    {% if redirect_field_value %}
+    <input type="hidden" name="{{ redirect_field_name }}" value="{{ redirect_field_value }}"/>
+    {% endif %}
+    <button type="submit">Sign Out</button>
+</form>
```

Comment:
why do you need a form here? Can you just have this bre a button or a href link to /signout/ ? 

Just curious...

Reply:
That was just a shortcut because I was logging in and out a bunch testing stuff. If you send a psot request it directly signs you out instead of requiring another page load.